admin_guide/managing_networking.adoc

```diff
@@ -1201,3 +1201,57 @@ objects:
             name: default
 ...
 ----
+
+[[admin-guide-enabling-hsts]]
+== Enabling HTTP Strict Transport Security
+
+HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
+requests that traffic to the host be HTTPS. This is useful for ensuring secure
+interactions with websites, or to offer secure application for the user's
+benefit.
+
+The default behavior is to drop any HTTP requests that require HTTPS. Enabling
+HSTS lets you set up a redirect so that the first time a client requests the
```

Comment:
Not quite.  All it does is add the header to https responses.  You can set the insecureEdgeTerminationPolicy to redirect to send http to https.

BUT if a browser has received an HSTS header from the site in the past, it will not send http traffic to the site and instead flip to https.  This part is what the change controls.

Reply:
@knobunc The client is not required to support hsts. Most clients do respect it and will send https.

Reply:
@knobunc @pecameron Using the `insecureEdgeTerminationPolicy` value doesn't seem to match up with the rest of the section. Is this an alternative to what's here?

Reply:
I have:

"You can use the `insecureEdgeTerminationPolicy`
value in a route to redirect to send HTTP to HTTPS. However, when HSTS is
enabled, the client changes all requests from the HTTP URL to HTTPS before the
request is sent, eliminating the need for a redirect."